CSM Front-end added.

AmayaNW opened the pull request.

AmayaNW committed:
CSM Front-end added.

AmayaNW committed:
_

AmayaNW committed:
...

Dushanee commented:
This branch is out-of-date with the base branch
Merge the latest changes from main into this branch.

AmayaNW committed:
...

AmayaNW committed:
--

AmayaNW committed:
2/15

AmayaNW committed:
..

AmayaNW committed:
..

AmayaNW committed:
structure changed.

AmayaNW committed:
bxcugd

AmayaNW committed:
asc

AmayaNW committed:
*

AmayaNW committed:
Merge branch 'main' into csm

AmayaNW committed:
committing to csm.

AmayaNW reviewed (commented):
Changes in csm.

AmayaNW committed:
New commit.

AmayaNW committed:
committed.

AmayaNW committed:
new nav-bar

AmayaNW committed:
Update navbar.css

AmayaNW committed:
Update home.css

AmayaNW committed:
cards

AmayaNW committed:
whole rating and feedback for the system.

AmayaNW committed:
Create submit-article.php

AmayaNW committed:
_

AmayaNW committed:
dashboard updated.

AmayaNW committed:
Merge branch 'main' into csm

AmayaNW committed:
system's average rating calculation done.

AmayaNW committed:
logout for csm

AmayaNW committed:
Privacy policy (System's terms and conditions) added.

AmayaNW committed:
Updates added.

AmayaNW committed:
Forum messages viewing section

AmayaNW committed:
Update messages.php

AmayaNW committed:
Added the following.

View of service provider's ratings and feedback for the whole system

View of service providers's forum messages

AmayaNW committed:
Added and updated.

Rating and feedback form on the whole system for service provider added.

backend to save the Rating and feedback on the whole system from service provider added.

home.css updated.

AmayaNW committed:
'Help Center For Customer' added.

Should be displayed under the customer support from customer's end. Single page.

AmayaNW committed:
Create help_center_sp.php

Should be displayed under customer support from the service provider's end. Single page.

AmayaNW committed:
Merge branch 'main' into csm

AmayaNW commented:
Can anyone please review my code? I can't do Merge without that.

AmayaNW committed:
Create update_help_center.php

AmayaNW committed:
Create update_help_center.css

AmayaNW committed:
Create create_help_cards.php

AmayaNW committed:
Create view_helpTable_csm.php

AmayaNW committed:
Create create_faqs.php

AmayaNW committed:
Create create_faqs_sp.php

AmayaNW committed:
Create view_customer_faqs_table.php

AmayaNW committed:
Create view_sp_faqs_table.php

AmayaNW committed:
Create create_faqs_sp.php

AmayaNW committed:
Create create_faqs.php

AmayaNW committed:
Update help_center_sp.php

AmayaNW committed:
Update help_center.php

AmayaNW committed:
Folder changed.

AmayaNW committed:
Update nav-bar.php

AmayaNW committed:
Update update_help_center.php

AmayaNW committed:
Update addtocart.php

This is the shopping cart.

AmayaNW committed:
Create help_center_form.php

AmayaNW committed:
Update help_center.php

AmayaNW committed:
Shopping Cart

AmayaNW committed:
Shopping cart
